### PR TITLE
use package.json for defaults

### PR DESCRIPTION
**Pull request by jarofghosts**
hey! so i tried my best to match your style and i've never used async before, so this might not be 100% but it seems to be working on my end.

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -14,18 +15,28 @@ var config=require("../.config.json");
...
+              package = require(path.join(process.cwd(), 'package.json'))
+              callback(null, package)
+            } catch(e) {
+              callback(null, {})
+            }
+        },
+        function (package, callback){
             exec('git config --get user.name', function (err, stdout, stderr) {
-                if (err) throw er;
+                if (err) throw err;
```
Comment (Daniel1of1) [inline bin/cli.js:28]: thanks for the correction :S

File: bin/cli.js [hunk 2]
```diff
@@ -14,18 +15,28 @@ var config=require("../.config.json");
...
         },
-        function (gitUser, callback){
+        function (gitUser, package, callback){
+          package.author = package.author || {}
             var defaults={
                 year : config.year || new Date().getFullYear(),
-                organization : config.organization || gitUser || process.env.USER, 
-                project :  config.project || process.cwd().split("/").slice(-1)[0],
-                licenseType : config.licenseType || 'bsd3'
+                organization : config.organization || package.author.name || gitUser || process.env.USER,
```
Comment (Daniel1of1) [inline bin/cli.js]: What do you think about `package.author.name || config.organization`... instead. So it reflects a little bit more what git does with `.gitignore` ?

I don't mind either way.

**Comment by Daniel1of1** [comment]
Very sorry for such a late response. I've been a little busy. Back to github though so expect my replies to be more prompt :)

Check the inline comments I'll merge as soon as you check them out.

File: bin/cli.js (hunk 2 above)
Comment (jarofghosts) [inline bin/cli.js]: so you would rather the `package.json` information overrides the command line option? i don't care either way for my use case, but it seems like that may be confusing considering it operates the opposite way for everything else.

Comment (Daniel1of1) [inline bin/cli.js]: It won't override cli arguments. Cli arg will always take priority in [cliArgsParser](https://github.com/licenses/lice-js/blob/master/bin/cliArgsParser.js#L47-L49).

Comment (jarofghosts) [inline bin/cli.js]: ah, sorry i forgot all about that. pushed the change!

**Comment by Daniel1of1** [comment]
awesome thanks for the PR

**Pull request merged**